### ISSUE TITLE
grub-mkconfig not detecting initrd.img

### ISSUE THREAD

**Issue by Aldair502** [venomlinux/ports#7664]
Idk why this happens but after gen a image with mkinitramfs it gens the image and it works just fine. but grub can't detect it so i need to add it manually

this happend on linux-libre 6.6.5

**Comment by Visone-Selektah** [comment]
What name have the image? it should be `initrd-venom.img`
Are you using the grub pkg of repos?

**Comment by Aldair502** [comment]
> What name have the image? it should be `initrd-venom.img` Are you using the grub pkg of repos?

yeah initrd-venom.img and im using the grub from this repo. also like i said it just happens after using grub-mkconfig in /boot/grub/grub.cfg
